CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `PlutusV2.AlwaysSucceeds` example, for testing PlutusV2 scripts. ([#947](https://github.com/Plutonomicon/cardano-transaction-lib/pull/947))
 - `InlineDatum` example, for testing inline datum constraints. ([#931](https://github.com/Plutonomicon/cardano-transaction-lib/pull/931))
 - `DatumPresence` data type, which tags paying constraints that accept datum, to mark whether the datum should be inline or hashed in the transaction output. ([#931](https://github.com/Plutonomicon/cardano-transaction-lib/pull/931))
+- Added conversion functions between `PlutusData` and `CborBytes` ([#1001](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1001))
```

Comment:
Please add names of the functions, with module names: `Contract.PlutusData.serializeData` and `Contract.PlutusData.deserializeData`